src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/RazorCompletionEndpoint.cs

```diff
@@ -78,11 +75,16 @@ public TextDocumentIdentifier GetTextDocumentIdentifier(CompletionParams request
 
         var correlationId = Guid.NewGuid();
         using var _ = _telemetryReporter?.TrackLspRequest(Methods.TextDocumentCompletionName, LanguageServerConstants.RazorLanguageServerName, correlationId);
+        var razorCompletionOptions = new RazorCompletionOptions(
+            SnippetsSupported: true,
```

Comment:
It's set to false here 

https://github.com/dotnet/razor/blob/45b65b3eb3ba60c010e1bf8b4e021e2a8a85c26e/src/Razor/src/Microsoft.VisualStudio.LegacyEditor.Razor/Completion/RazorDirectiveAttributeCompletionSource.cs#L81

though it's a bit hard to see because default value for the RazorCompletionOptions gets used there

https://github.com/dotnet/razor/blob/45b65b3eb3ba60c010e1bf8b4e021e2a8a85c26e/src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Completion/RazorCompletionContext.cs#L16

had the same question myself and found this after digging around :)